experimental/dds/tree2/src/test/utils.ts

```diff
@@ -309,12 +310,13 @@ export class TestTreeProvider {
 	 * was set to true when calling the create() method.
 	 * @returns void after a summary has been resolved. May be called multiple times.
 	 */
-	public async summarize(): Promise<void> {
+	public async summarize(): Promise<SummaryInfo> {
 		assert(
 			this.summarizer !== undefined,
 			"can't summarize because summarizeOnDemand was not set to true.",
 		);
-		await summarizeNow(this.summarizer, "TestTreeProvider");
+		const summary = await summarizeNow(this.summarizer, "TestTreeProvider");
```

Comment:
better to do `return await summarizeNow(this.summarizer, "TestTreeProvider")` here. The local variable name `summary` isn't really accurate.

Reply:
updated to return `summarizeNow(this.summarizer, "TestTreeProvider")` directly instead of first storing it into local variable